### ISSUE TITLE
Add ability to override default css per user (config related)

### ISSUE THREAD

**Issue by qwertygc** [wallabag/wallabag#1278]
Hello!
Just a idea : add a textarea for custom design on css.
The textarea modify the file custom.css, that by inheritance, modify the main theme.

**Comment by j0k3r** [comment]
You mean something that should be saved in the database per user and that will override the current theme?

**Comment by qwertygc** [comment]
Yes

**Comment by onbelangrijk** [comment]
I would very much appreciate this! 
The new theme with its blueish colours combined with red and white... well, a matter of taste ;)
I always loved the baggy theme because of its clarity and black/white design.

**Comment by j0k3r** [comment]
@onbelangrijk baggy will still be available on the v2, it won't be the default one (at least for now)

**Comment by onbelangrijk** [comment]
Thanks, good to hear, I wasn't aware of this.

Am 05.08.2015 um 11:11 schrieb Jeremy Benoist:

**Comment by qwertygc** [comment]
It's really simple to implante. In fact, simply to call the user.css after the theme css file. We can imagine a simple textarea write the theme.css.

I dont' know symphony and not the wallabag code, but i can write a PROOF code : 
```
if(isset($_POST['theme'])) {
file_put_contents('user.css', $_POST['theme']));
}
<textarea name="theme"><?php echo file_get_contents('user.css'); ?></textarea>
```

And in the template file : 
```
        <link rel="stylesheet" href="/themes/baggy/css/main.css" media="all">
        <link rel="stylesheet" href="/themes/baggy/css/user.css" media="all">
```

**Comment by j0k3r** [comment]
@qwertygc I know it's simple :wink:

**Comment by nicosomb** [comment]
Fixed width #4151
